### PR TITLE
refactor(octon): hard-cut execution roles

### PR DESCRIPTION
**Pull request by jamesryancooper**
## What
Hard-cuts Octon from the legacy agency ontology to the execution-role
subsystem and rewires the live authority, runtime, validator, and CI paths to
that model.

## Why
The repository still carried active `framework/agency/**`, `actor_ref`, and
`agent-augmented` execution semantics even though the target state is
run-first, execution-role-governed, and support-bounded. This PR makes the
repository match the execution-role hard-cutover packet end-to-end.

No-Issue: proposal packet implementation and migration closeout

## How
Moves the live role subsystem to `framework/execution-roles/**`, replaces
legacy role nouns with orchestrator/specialist/verifier/composition profile,
adds v3 runtime request/receipt schemas plus runtime events, narrows the live
support envelope to the admitted repo-shell/CI tuples, and updates validators,
workflows, templates, and CI to the new target state.

Rejected alternatives: compatibility aliases, dual role paths, keeping
browser/API live without runtime proof, and preserving `workflow run` as a
consequential compatibility lane.

## Profile Selection Receipt
- Semantic version source(s): `.octon/octon.yml`, workspace charter pair
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): governance/control-plane migration; branch-wide atomic replacement of live role ontology; rollback is whole-change revert only; browser/API support remains stage-only until runtime-real proof exists
- Tie-break status: no tie-break required
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan
- Workstreams and scope:
  - replace `framework/agency/**` and `instance/agency/runtime/**` with execution-role surfaces
  - rewire ingress, manifests, overlays, CI, bootstrap, scaffolding, and methodology docs
  - update runtime contracts and kernel/authority code to `execution_role_ref`, `role-mediated`, and the v3 schema set
  - narrow support-target truth to the admitted live tuples and restage browser/API as non-live
  - add hard-cut validation and promotion evidence
- Public interfaces/types/contracts affected:
  - execution-role registry and role contracts
  - execution request/receipt schema surface
  - support-target declarations and effective support matrix
  - workflow/capability classification terms used by runtime and CI
- Test scenarios:
  - hard-cut validator floor
  - support/capability/proof validators
  - `cargo test -p octon_authority_engine -p octon_kernel`
- Assumptions/defaults:
  - historical lineage artifacts remain retained history and are not back-edited
  - browser/API remain stage-only because runtime services and proof are not fully promoted

## Impact Map (code, tests, docs, contracts)
- Code:
  - `.octon/framework/engine/runtime/crates/{authority_engine,kernel}/**`
  - `.octon/framework/orchestration/runtime/_ops/scripts/write-run.sh`
- Tests/validators/CI:
  - `.octon/framework/assurance/**`
  - `.github/workflows/**`
  - `.octon/framework/execution-roles/_ops/scripts/validate/validate-execution-roles.sh`
- Docs/bootstrap/templates:
  - `.octon/README.md`
  - `.octon/instance/{ingress,bootstrap}/**`
  - `.octon/framework/{cognition,scaffolding}/**`
- Contracts/policies:
  - `.octon/framework/constitution/**`
  - `.octon/framework/engine/runtime/spec/**`
  - `.octon/instance/governance/{policies,support-targets.yml,support-target-admissions/**}`
  - `.octon/generated/effective/governance/support-target-matrix.yml`

## Compliance Receipt
- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations
none

## Tradeoffs
This is intentionally a large atomic migration. The branch retires the old
live role path in one cut, so review cost is higher than a stacked series, but
it avoids the dual-ontology state the packet explicitly rejects.

Browser/API remain stage-only rather than live. That keeps the support claim
truthful, but it also means the PR does not expand the admitted support
universe.

## Testing
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-role-hard-cutover.sh`
- `bash .octon/framework/execution-roles/_ops/scripts/validate/validate-execution-roles.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-ingress-manifest-parity.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-harness-structure.sh`
- `bash .octon/framework/assurance/scripts/validate-support-target-canonicality.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/validate-support-target-normalization.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/validate-capability-truthfulness.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-support-dossier-parity.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-proof-plane-coverage.sh`
- `cargo test -p octon_authority_engine -p octon_kernel`

## Rollout
Internal harness cutover. No feature-flag rollout. Merge when checks are green;
rollback is revert of the squash commit if post-merge issues appear.

## Convivial Purpose Check
- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric
- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert the cutover commit as one atomic change
- Rollback handle: revert merged squash commit on `main`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model
- OpenAPI/JSON-Schema changes:
  - execution request/receipt v3
  - runtime event v1
  - support-target schema hardening
  - execution-role registry/contracts
- Threat model update/link:
  - support envelope narrowed to admitted tuples only; browser/API remain stage-only until proof-backed

## Observability and Performance
- Traces/logs/metrics for changed flows:
  - runtime event schema introduced for canonical execution lifecycle events
  - proof/assurance validators updated to the execution-role model
- Representative traces for high risk changes:
  - existing retained run evidence remains referenced by proof-plane coverage validation
- Performance or bundle impact:
  - no material runtime bundle expansion expected beyond schema/validator/doc churn

## License and Provenance
- New dependencies and licenses: none
- Generated code/templates provenance notes:
  - committed proposal packet remains non-authoritative historical lineage
  - retained promotion evidence added under `.octon/state/evidence/validation/publication/**`

## Checklist
- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] Reviewer feedback addressed with fix, commit, push, reply; reviewer-owned threads left for reviewer or maintainer confirmation

## Screenshots / Notes
No UI changes.

### CODE REVIEW

File: .octon/framework/orchestration/runtime/workflows/audit/audit-design-proposal/workflow.yml [hunk 1]
```diff
@@ -165,7 +165,7 @@ stages:
     allowed_executor_profiles:
     - scoped_repo_mutation
 - id: design-red-team
-  asset: stages/04-design-red-team.md
+  asset: stages/04-design-red-PROFILE.md
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/orchestration/runtime/workflows/audit/audit-design-proposal/workflow.yml]: Updated the audit-design-proposal workflow and README to reference the existing `stages/04-design-red-team.md` asset. The workflow validator now passes locally against the corrected path.

File: .octon/framework/assurance/runtime/_ops/scripts/validate-phase6-simplification-deletion.sh [hunk 2]
```diff
@@ -102,16 +102,16 @@ main() {
...
-  require_text '.octon/framework/agency/runtime/agents/orchestrator/AGENT.md' "$INGRESS" "instance ingress points at orchestrator contract"
+  forbid_text 'framework/execution-roles/governance/CONSTITUTION.md' "$INGRESS" "instance ingress excludes agency CONSTITUTION.md from kernel path"
+  require_text '.octon/framework/execution-roles/runtime/orchestrator/ROLE.md' "$INGRESS" "instance ingress points at orchestrator contract"
   require_text 'historical-shim' "$CONTRACT_REGISTRY" "contract registry demotes duplicate constitutional shims"
 
-  forbid_text 'SOUL.md' "$OCTON_DIR/framework/agency/runtime/agents/orchestrator/AGENT.md" "orchestrator contract no longer depends on SOUL.md"
-  forbid_text 'SOUL.md' "$OCTON_DIR/framework/agency/runtime/agents/verifier/AGENT.md" "verifier contract no longer depends on SOUL.md"
+  forbid_text 'SOUL.md' "$OCTON_DIR/framework/execution-roles/runtime/orchestrator/ROLE.md" "orchestrator contract no longer depends on SOUL.md"
+  forbid_text 'SOUL.md' "$OCTON_DIR/framework/execution-roles/runtime/agents/verifier/AGENT.md" "verifier contract no longer depends on SOUL.md"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/validate-phase6-simplification-deletion.sh]: Updated the Phase 6 validator to check the live verifier contract at `runtime/verifiers/independent-verifier/VERIFIER.md` and the renamed `execution-role-validate.yml` workflow. The local Phase 6 validator now passes with the execution-role layout.
